### PR TITLE
Add Math Notes

### PR DESCRIPTION
**Pull request by LemonPi**
Syntax highlighting for taking math notes

### CODE REVIEW

**Comment by LemonPi** [comment]
Added release 1.0.0 tag.
How can I retrigger packagecontrol-bot?

**Comment by FichteFoll** [comment]
@LemonPi for now you can close and re-open the PR. Please address the reported warning, then we can merge.

**Comment by LemonPi** [comment]
@FichteFoll Thanks. The [help wiki](https://github.com/packagecontrol/st_package_reviewer/wiki/Package-checks) says that 

> In your sublime_text version selector, you should write that the package requires at least build 3092

Can you elaborate what 
> sublime_text version selector 

is (maybe by editing the help wiki)? I have no idea what that refers to after 30 minutes of search.

File: repository/m.json [hunk 1]
```diff
@@ -751,6 +751,17 @@
 			]
 		},
 		{
+			"name": "Math Notes",
+			"details": "https://github.com/LemonPi/math-notes",
+			"labels": ["language syntax", "documentation", "notes"],
+			"releases": [
+				{
+					"sublime_text": "*",
```
Comment (FichteFoll) [inline repository/m.json]: ^ It's this line.

**Pull request merged**